src/test/suite/attribute.test.ts

```diff
@@ -0,0 +1,92 @@
+import * as assert from "assert";
+import { beforeEach, it, afterEach, describe } from "mocha";
+import * as sinon from "sinon";
+import { reset } from "ts-mockito";
+import { Range } from "vscode";
+import {
+  activate,
+  clearUnsavedChanges,
+  getDocUri,
+  sleep,
+} from "./utils/helper";
+import {
+  isProcessReadyForTest,
+  readLineMock,
+  requestResponseItems,
+  stdinMock,
+  stdoutMock,
+} from "../../binary/mockedRunProcess";
+import { resetBinaryForTesting } from "../../binary/requests/requests";
+import {
+  makeAChange,
+  mockAutocomplete,
+  moveToActivePosition,
+  triggerInline,
+} from "./utils/completion.utils";
+import {
+  anAutocompleteResponse,
+  INLINE_NEW_PREFIX,
+  INLINE_PREFIX,
+  SINGLE_CHANGE_CHARACTER,
+} from "./utils/testData";
+import * as vscodeApi from "../../vscode.api";
+
+describe("Should show attribution item on inline suggestion", () => {
+  const docUri = getDocUri("completion.txt");
```

Comment:
Maybe create another file for this test? I don't think it's a good idea sharing files between tests